CDS/src/org/icpc/tools/cds/video/containers/HLSHandler.java

```diff
@@ -29,16 +31,20 @@
  */
 // TODO - cache HLS m3u8
 // TODO - handle HLS query params
-// TODO - no hardcoded stream
-// TODO - multiplexing
-// TODO - cache cleanup
+// TODO - multiplexing m3u8
+// TODO - switch gap.mp4 to using #EXT-X-GAP
```

Comment:
This is not a todo anymore is it?

Reply:
Yeah, I missed that. I have the next PR almost ready, I'll fix it there.